src/plone/restapi/serializer/__init__.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+from Acquisition import aq_inner
+from Acquisition import aq_parent
+from Products.CMFCore.interfaces import IContentish
+from Products.CMFCore.interfaces import IFolderish
+from Products.CMFPlone.interfaces import IPloneSiteRoot
+
+from plone.app.contenttypes.interfaces import ICollection
+from plone.restapi.interfaces import ISerializeToJson
+from plone.restapi.interfaces import IFieldSerializer
+from plone.restapi.serializer.converters import json_compatible
+from plone.dexterity.utils import iterSchemata
+
+from zope.site.hooks import getSite
+from zope.interface import implementer
+from zope.component import adapter
+from zope.component import getMultiAdapter
+from zope.component import queryMultiAdapter
+from zope.schema import getFields
+
+
+def lookup_field_serializer(context, field):
+    serializer = queryMultiAdapter((field, context, context.REQUEST),
+                                   IFieldSerializer,
+                                   name=field.interface.__identifier__)
```

Comment:
I agree. I see "specific field" and "entire field type" as the two main customization use cases. If it turns out people actually need something in between, we can always add that later.